.travis.yml

```diff
@@ -32,3 +35,20 @@ matrix:
 
   allow_failures:
     - env: OPTS="--HEAD"
+
+before_deploy:
+  - brew install jq
+  - brew bottle rnp --json
+  - source ci/read_bottle_metadata.sh
+  - ci/build_bintray_descriptor.rb
+
+deploy:
```

Comment:
User is `rnp-ci`. Token can be accessed via GitHub Actions `{{ secret.BINTRAY_CI_TOKEN }}`.